src/main/java/me/zeroeightsix/kami/module/modules/movement/ElytraReplace.kt

```diff
@@ -22,25 +23,33 @@ class ElytraReplace : Module() {
     private val threshold = register(Settings.integerBuilder("Broken %").withRange(1, 100).withValue(7).build())
     private val inventoryMode = register(Settings.b("Inventory", false))
     private val elytraFlightCheck = register(Settings.b("ElytraFlight Check", true))
+    private val logToChat = register(Settings.booleanBuilder("Log Missing Replacement").withValue(false).build())
+    private val playSound = register(Settings.booleanBuilder("Play Sound").withValue(false).withVisibility { logToChat.value }.build())
+    private val logThreshold = register(Settings.integerBuilder("Log Threshold").withRange(1, 10).withValue(3).withVisibility { logToChat.value }.build())
     private var currentlyMovingElytra = false
     private var currentlyMovingChestplate = false
     private var elytraCount = 0
+    private var chestplateCount = 0
 
     override fun onUpdate() {
         if (!inventoryMode.value && mc.currentScreen is GuiContainer) {
             return
         }
 
-        elytraCount = InfoOverlay.getItems(Items.ELYTRA) + InfoOverlay.getArmor(Items.ELYTRA)
-        val chestplateCount = InfoOverlay.getItems(Items.DIAMOND_CHESTPLATE) + InfoOverlay.getArmor(Items.DIAMOND_CHESTPLATE)
+        elytraCount = 0
+        for (i in 0..44) {
```

Comment:
where is the number 44 from?

Reply:
I honestly am not quite sure what the 44 inventory slots would be, but this is what the module uses to look for elytras and chestplates later in the code, so I merely changed that a bit to do what I wanted. According to the file history, iBuyMountainDew initially implemented a for loop with i < 45, and you added the 0..44 when you kotlinized the module.

Reply:
Oh I see. I was confused but if it starts at 0, it should be fine. Thanks!